broker-node/config/segment.php

```diff
@@ -24,6 +24,5 @@
     |
     */
 
-    'write_key' => env('SEGMENT_WRITE_KEY'),
-
+    'write_key' => env('SrQ0wxvc7jp2XDjZiEJTrkLAo4FC2XdD'),
```

Comment:
Shouldn't be committing this key into Github. This will read `SEGMENT_WRITE_KEY` from `broker-node/.env`